packages/editor/src/index.ts

```diff
@@ -61,7 +61,7 @@ export default createPrompt<string, EditorConfig>((config, done, rl) => {
     }
   });
 
-  if (!(config.waitUserInput ?? true) && firstRender && rl != undefined) {
+  if (!(config.waitUserInput ?? true) && firstRender && rl !== undefined) {
```

Comment:
```suggestion
  if (config.waitUserInput != null && firstRender && rl !== undefined) {
```

Reply:
What about the code below so the user can also pass `true` (default) and the user is still required to press `<enter>`.

```ts
if (config.waitUserInput != null && config.waitUserInput === false && firstRender && rl !== undefined) {
```

Thanks also for the other suggestions/changes, I will look into those and try to fix them.

Reply:
Also works, but since it's the default, maybe we could just type as `waitUserInput?: false`. No big difference either way; just don't like the double conditions `!(a ?? B)` (or other similar forms)